### DISCUSSION TITLE
Recent updates on our website

### DISCUSSION THREAD

**Discussion by rholzhause** [techtriq/qibb#1] [Announcements]
### 🌐 Recent updates on our website qibb.com
We’ve just relaunched the **qibb website** with a brand new structure, content and key messaging. 
As a result of our workover, there’s a leaner site with less pages, reduced content, and a more intuitive user guidance.

You can check out the new content here: https://www.qibb.com

### 📚 Public documentation about qibb
In addition, we have launched a new [public documentation](https://docs.qibb.com), containing 130+ pages of common actions and best practices using the qibb platform as well as our prebuilt use cases: Cloud Editing, Cloud Archiving & Cloud Playout.

Would really love to hear your thoughts and reactions about the recent updates.
_Cheers!_

**Comment by ghost** [comment]
We would be glad if you could review any part of the documentation. Please let me know here or any other direct channel to me (e.g. n.horvath@techtriq.com) if you have any comment or feedback! Thanks!
